rskj-core/src/main/java/co/rsk/rpc/Web3EthModule.java

```diff
@@ -105,6 +105,10 @@ default String eth_sendTransaction(Web3.CallArguments args) {
 
     BlockResultDTO eth_getBlockByNumber(String bnOrId, Boolean fullTransactionObjects) throws Exception;
 
+    String eth_getRawBlockHeaderByHash(String blockHash) throws Exception;
```

Comment:
I think all this shouldn't go under the `eth_` namespace as it may imply these are part of the standard. What about creating a new module for handling `ext_`?